package.json

```diff
@@ -9,7 +9,9 @@
     "lint-fix": "npm run eslint -- --fix && npm run prettier -- --write",
     "eslint": "eslint './**/*.js'",
     "prettier": "prettier './**/*.js' './**/*.sol' './**/*.md'",
-    "test": "lerna run --stream --concurrency=1 test",
+    "test": "yarn test-no-concurrent && yarn test-concurrent",
+    "test-no-concurrent": "lerna run --stream --concurrency=1 test --scope @uma/serverless-orchestration --scope @uma/affiliates",
+    "test-concurrent": "lerna run --stream --concurrency=4 test --ignore @uma/serverless-orchestration --ignore @uma/affiliates",
```

Comment:
Why can these not be run in parallel?

Reply:
These two packages, in particular, rely on some block number/time manipulation that seems to break within the concurrent context. I still need to test it out more but for now, this seems to in a stable way.

Reply:
Makes sense.